src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -100,6 +100,21 @@ public static int createMonitoringTopicIfNotExists(String zkUrl, String topic, i
   }
 
   /**
+   * @param zkUrl zookeeper connection url
```

Comment:
Yes I can see it now. I didn't find this by doing Ctrl-F and search for getBrokerCount() in the web page probably because the MirrorPipelineMonitor didn't load successfully.